android/README.md

```diff
@@ -0,0 +1,72 @@
+# IronOxide-Android
+
+[![javadoc.io](https://javadoc.io/badge2/com.ironcorelabs/ironoxide-java/javadoc.io.svg)](https://javadoc.io/doc/com.ironcorelabs/ironoxide-java)
+
+## Using IronOxide-Android
+
+In order to make calls to the IronCore service, you must add a dependency for ironoxide-android, load the library, and provide authentication.
+
+### Adding a dependency
+
+Add the following to your module's "build.gradle":
+
+```java
+dependencies {
+    implementation 'com.ironcorelabs:ironoxide-android:<VERSION>@aar'
+}
+```
+
+### Loading the library
+
+To load the `ironoxide-android` library in Java, use the following:
+
+```java
+java.lang.System.loadLibrary("ironoxide_android");
+```
+
+### Authenticating
+
+To make calls, you must create a project and segment in the IronCore Admin Console. Once created, you can use a tool such as [ironoxide-cli](https://github.com/IronCoreLabs/ironoxide-cli) to create users and devices. You can then read in a device with the `DeviceContext.fromJsonString()` function and use it to initialize an IronOxide instance. An example of this is available in the [Example Application](/android/examples/Example_Application/) in "MainActivity.java".
```

Comment:
I like this pointer to `ironoxide-cli` to get started quickly, but it reads like we recommend that as a long term solution, which we don't. I don't think we need to expand a ton here, but maybe we should qualify our `ironoxide-cli` pointer a little.

Reply:
I see what you mean. Until we have the official JWT docs posted, I'll include a link to ironoxide's JWTClaims struct to help show what form it should take.